src/LLVM/Pretty.hs

```diff
@@ -30,15 +28,21 @@ import qualified LLVM.AST.Float as F
 import LLVM.AST.FunctionAttribute
 
 import Data.String
+import qualified Data.ByteString.Short as SBF
+import qualified Data.ByteString.Char8 as BF
+import Data.ByteString.Internal(w2c)
 
+import Data.Text.Format
 import Text.Printf
-import Data.Text.Lazy (Text, pack, unpack)
```

Comment:
Why are you changing the Text type from lazy to strict? I don’t think it matters a whole lot here but lazy seems more appropriate here.